src/contracts/core/AVSDirectoryStorage.sol

```diff
@@ -50,6 +50,29 @@ abstract contract AVSDirectoryStorage is IAVSDirectory {
     /// @notice Mapping: avs => operatorSetId => Total operators within the given operator set.
     mapping(address => mapping(uint32 => uint256)) public operatorSetMemberCount;
 
+    /// @notice Mapping: operator => avs => operatorSetId => operator registration status
+    mapping(address => mapping(address => mapping(uint32 => OperatorSetRegistrationStatus))) public operatorSetStatus;
```

Comment:
why do we add this?

Reply:
We need a last deregistered timestamp for slashability. See isOperatorSlashable view func